localization/english/wc_traits_l_english.yml

```diff
@@ -161,16 +161,19 @@
  trait_creature_green_dragon_character_desc:0 "$trait_creature_green_dragon_desc$"
  
  trait_creature_black_dragon:0 "Black Dragon"
  trait_creature_black_dragon_desc:0 "Black dragons are large creatures that are carnivorous. They spend their time hunting, as well as watching the various wars around the world, and can often be found torturing prisoners, animals, or anything else unfortunate enough to fall into their care. A black dragon is a large, thickly-built creature with a massive chest and a long, sinuous neck. Its hide is a rich, lustrous black with large scales that are supple yet hard as rock. A crest made of peaked bone spikes runs from its head and down its back, and bristles when the great beast grows angry. Its fangs shine a coppery-red, and deep inside its mouth a constant sulpurous light emits a faint glow. Most civilized races say that black dragons have a cruel, callous look to their serpentine faces. Black dragons do not shed individual scales, but rather molt once a year, usually by spending a week resting within a lava flow of an active volcano, which peels off and consumes their discarded skin. The skin of a black dragon nearing its molt grows dull and gray. Most black dragons appear with their horns and wings in a shade of orange to brown, but a few females appear with purplish wings and horns."
+ trait_creature_black_dragon_character_desc:0 "$trait_creature_black_dragon_desc$"
  
  trait_creature_bronze_dragon:0 "Bronze Dragon"
  trait_creature_bronze_dragon_desc:0 "Bronze dragons are smaller and more agile than dragons of the other flights, and they appear to be built for speed rather than stamina. Their bodies are thin and dexterous, with scales almost metallic in appearance. The quality of their scales is such that they are sometimes mistaken for constructs or detailed statuary. Bronze dragons are fastidious, keeping their mesh-like scales clean and shining at all times. The dragon is long and slender, its scales glinting with a metallic sheen. A spiky bone ridge runs down its back and forms a wicked point at the end of its tail. Its form shimmers from golden yellow to rich orange in the sunlight as it darts through the air. Bronze dragons prefer to use their spell-like abilities and breath weapon to confuse and scatter enemies. Afterward, they approach foes singly, swooping from the air or fighting on the ground depending on which approach offers the best tactical advantage. Bronzes can also be very patient, watching from afar until a target lets down its guard."
+ trait_creature_bronze_dragon_character_desc:0 "$trait_creature_bronze_dragon_desc$"
```

Comment:
The other creature trait character desc are prefixed with "This character is a (race)"

Reply:
I just added the line we had for some dragons to the other ones here...

Reply:
However, fixed.